Gems/Multiplayer/Code/Source/NetworkInput/NetworkInput.cpp

```diff
@@ -86,6 +86,39 @@ namespace Multiplayer
         return m_hostBlendFactor;
     }
 
+    void NetworkInput::LogComponentInputDelta() const
+    {
+        for (uint16_t i = 0; i < m_componentInputs.size(); ++i)
+        {
+            //AZLOG_WARN(m_componentInputs[i].get()->GetInputDeltaLog().c_str());
```

Comment:
minor: Avoid committing commented out code. Consider if function has any value as is